code/modules/clothing/head/helmet.dm

```diff
@@ -58,16 +58,24 @@
 	body_parts_covered = HEAD|FACE|EARS
 	siemens_coefficient = 1
 
-/obj/item/clothing/head/helmet/tactical
-	name = "tactical helmet"
+/obj/item/clothing/head/helmet/NT
+	name = "\improper NT helmet"
 	desc = "An armored helmet capable of being fitted with a multitude of attachments."
 	icon_state = "swathelm"
-
 	armor = list(melee = 62, bullet = 50, laser = 50,energy = 35, bomb = 10, bio = 2, rad = 0)
 	flags_inv = HIDEEARS
 	siemens_coefficient = 0.7
 	price_tag = 250
 
+/obj/item/clothing/head/helmet/merc
+	name = "tactical helmet"
+	desc = "A tan helmet made from advanced ceramic. Comfortable and robust."
+	icon_state = "merchelm"
+	item_flags = THICKMATERIAL | COVER_PREVENT_MANIPULATION
+	flags_inv = HIDEEARS|HIDEEYES|BLOCKHAIR
+	armor = list(melee = 65, bullet = 70, laser = 70, energy = 45, bomb = 40, bio = 0, rad = 0)
```

Comment:
What is his even, we needed the alternative to IH armor, not SWAT Armor Tesla Bugalo. Make it have the same stats IH helmet have.